bower.json

```
@@ -1,23 +1,24 @@
 {
   "name": "adapt-mediaWithBenefits",
-  "version": "0.0.1",
-  "framework": ">=3",
+  "version": "0.0.2",
+  "framework": ">=5",
```

Due to use of ES6 arrow functions
```suggestion
  "framework": ">=5.3",
```